demisto_sdk/commands/common/update_id_set.py

```diff
@@ -1298,20 +1344,20 @@ def process_jobs(file_path: str, print_logs: bool, marketplace: str = '') -> lis
     """
     result: List = []
     try:
-        if should_skip_item_by_mp(file_path, marketplace, {}, print_logs):
+        if should_skip_item_by_mp(file_path, marketplace, {}, packs=packs, print_logs=print_logs):
             return []
         if find_type(file_path) == FileType.JOB:
             if print_logs:
                 print(f'adding {file_path} to id_set')
-            result.append(get_job_data(file_path, print_logs))
+            result.append(get_job_data(file_path, packs=packs))
```

Comment:
Did we use to print twice?

Reply:
yes, it also appears in the calling function